.cicd/pipeline.yml

```diff
@@ -104,7 +104,7 @@ steps:
     timeout: ${TIMEOUT:-120}
     agents:
       - "queue=mac-anka-large-node-fleet"
-    skip: $SKIP_MOJAVE
+    skip: ${SKIP_MACOS_10_14}
```

Comment:
I would rather change everything else from `SKIP_MACOS_10_14` to `SKIP_MOJAVE` than to move these to the long and not human-friendly one.

Reply:
All the new branches (1.8.x and develop) are already the long ones, so this is actually consistent with the others and EOSIO.